storage/engine/db.cc

```diff
@@ -27,9 +27,9 @@
 #include "rocksdb/options.h"
 #include "rocksdb/table.h"
 #include "rocksdb/utilities/write_batch_with_index.h"
-#include "cockroach/proto/api.pb.h"
-#include "cockroach/proto/data.pb.h"
-#include "cockroach/proto/internal.pb.h"
+#include "proto/api.pb.h"
+#include "proto/data.pb.h"
+#include "proto/internal.pb.h"
```

Comment:
Remember I mentioned having to `cd` out of the repo? that's what gave the `cockroach/` prefix. Now that I'm not doing that, these import paths mirror the on-disk layout of the protos relative to the repository root.